### ISSUE TITLE
Is this mondule still maintained?

### ISSUE THREAD

**Issue by vanthome** [doxout/recluster#53]
As a user of this module, I'm asking: Is this module still maintained?
thx

**Comment by spion** [comment]
Not really, no. There is very little interest for it nowadays as other tools (e.g. container orchestraction via kubernetes or / API gateway for lambdas) are better suited to fill the gaps.

**Comment by vanthome** [comment]
hmm, I'm not sure whether I agree with this, I run a lot of node based micro services and I do use clustering within a single container/ service. I think it's a good way for light weight of scaling node services as you still benefit from the single memory space.

**Comment by spion** [comment]
The memory space of cluster forks is not shared, even though the APIs make it sound like it is; it doesn't actually use fork syscalls (or at least it didn't last time I checked)
